backend/TreeOfAKind.API/Trees/AddTreePhotoRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Http;
+
+namespace TreeOfAKind.API.Trees
+{
+    public class AddTreePhotoRequest
+    {
+        [Required] public Guid TreeId { get; set; }
+        [Required] public IFormFile Image { set; get; }
```

Comment:
`set; get;`

Reply:
this one is the form file request media type?

Reply:
Yes.
And the request must contain body `from-form`